src/modules/Elsa.EntityFrameworkCore.MySql/Elsa.EntityFrameworkCore.MySql.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk">
+    
+    <Import Project="..\..\..\common.props" />
+    <Import Project="..\..\..\configureawait.props" />
+    
+    <PropertyGroup>
+        <TargetFrameworks>net6.0;net7.0</TargetFrameworks>
+        <Description>
+            Provides MySql implementation and migrations for various modules.
+        </Description>
+        <PackageTags>elsa module persistence efcore MySql</PackageTags>

```

Comment:
Please use lowercase tags for consistency with the other tags.

Reply:
sure, thanks!